source/swap/test/unit.js

```diff
@@ -141,6 +155,10 @@ describe('Swap Unit Tests', () => {
       tokenKinds.ERC20,
       erc20Handler.address
     )
+    // await transferHandlerRegistry.addTransferHandler(
```

Comment:
How does check work if the handler isn't added?

Reply:
The tests are wrong, I pushed an update to add the handler back but we need to review the tests. We have a similar issue with the unit test of erc1155.